update COCO format documentation

zmaslova opened the pull request:
<!---
Copyright (C) 2020-2021 Intel Corporation

SPDX-License-Identifier: MIT
-->

<!-- Raised an issue to propose your change (https://github.com/opencv/cvat/issues).
It will help avoiding duplication of efforts from multiple independent contributors.
Discuss your ideas with maintainers to be sure that changes will be approved and merged.
Read the [CONTRIBUTION](https://github.com/opencv/cvat/blob/develop/CONTRIBUTING.md)
guide. -->

<!-- Provide a general summary of your changes in the Title above -->

### Motivation and context
<!-- Why is this change required? What problem does it solve? If it fixes an open
issue, please link to the issue here. Describe your changes in detail, add
screenshots. -->

### How has this been tested?
<!-- Please describe in detail how you tested your changes.
Include details of your testing environment, and the tests you ran to
see how your change affects other areas of the code, etc. -->

### Checklist
<!-- Go over all the following points, and put an `x` in all the boxes that apply.
If an item isn't applicable by a reason then ~~explicitly strikethrough~~ the whole
line. If you don't do that github will show incorrect process for the pull request.
If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] I submit my changes into the `develop` branch
- [x] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

zhiltsov-max commented on cvat/apps/dataset_manager/formats/README.md:
```diff
@@ -536,8 +526,8 @@ archive.zip/
-
-*Note*: there is also a [support for COCO keypoints over Datumaro](https://github.com/openvinotoolkit/cvat/issues/2910#issuecomment-726077582)
+Support for COCO tasks via Datumaro is described [here](https://github.com/openvinotoolkit/datumaro/blob/develop/docs/formats/coco_user_manual.md#export-to-coco)
+For example, export COCO keypoints:
```
The link included useful information about logic of this export, I think, it should be either described here, or the link should be kept.

zmaslova replied:
the link is returned

zhiltsov-max reviewed (changes requested):
@azhavoro, do we need to sync with the docs site?

nmanovic commented:
@zmaslova , could you please update your PR? We have added a special site for our documentation: https://openvinotoolkit.github.io/cvat/docs/

zmaslova commented:
> @zmaslova , could you please update your PR? We have added a special site for our documentation: https://openvinotoolkit.github.io/cvat/docs/

Updated

nmanovic commented:
@zmaslova , there is a problem in the patch. Could you please fix it?

![image](https://user-images.githubusercontent.com/40690625/119293452-1d9ff480-bc5b-11eb-9ab1-be69cd8571a5.png)

zmaslova commented:
> @zmaslova , there is a problem in the patch. Could you please fix it?
> 
> ![image](https://user-images.githubusercontent.com/40690625/119293452-1d9ff480-bc5b-11eb-9ab1-be69cd8571a5.png)

Fixed. I also see issues in CHANGELOG.md, I suppose I shouldn't fix them
